lib/install/management/create.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/vmware/vic/pkg/vsphere/tasks"
 )
 
+const (
+	uploadRetryCount = 5
```

Comment:
Yes I discussed this with @mhagen-vmware and @emlin . We decided that in the end for now a simple for loop retry will suffice. This is also not meant to be a complex fix. this is simply going to allow vic-machine to recover from *minor* network blips. Nothing more.